api/src/queries/survey/survey-occurrence-queries.ts

```diff
@@ -561,30 +518,34 @@ export const getTemplateMethodologySpeciesSQL = (occurrenceId: number): SQLState
  * @param {string} inputKey
  * @return {*}  {(SQLStatement | null)}
  */
-export const getTemplateMethodologySpeciesIdSQL = (surveyId: number): SQLStatement | null => {
+export const getTemplateMethodologySpeciesRecordSQL = (
+  speciesId: number,
+  methodologyId: number,
+  templateId: number
+): SQLStatement | null => {
   defaultLog.debug({
-    label: 'getTemplateMethodologySpeciesIdSQLStatement',
+    label: 'getTemplateMethodologySpeciesRecordSQLStatement',
```

Comment:
nit: can you make this label, and the one in the log at the end of this function, match the function name?